components/inputs/input-date.js

```diff
@@ -252,7 +237,7 @@ class InputDate extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixin(
 				@d2l-dropdown-open="${this._handleDropdownOpen}"
 				@d2l-dropdown-focus-enter="${this._handleFocusTrapEnter}"
 				max-width="335"
-				min-height="${this._hiddenCalendarHeight}"
+				min-height="415"
```

Comment:
It looks (based on the comment above) that this assumes the root font-size is 20px; What happens when it's larger?  Is the experience any worse than before?

Reply:
Was just thinking, if needed, we could get the computed root `font-size` from the `html` element, and select from hard-coded heights.  18, 20, 22, and 24px being our discrete sizes.